#Load Data Set

In [5]:
import pandas as pd
import numpy as np


df = pd.read_csv("../data/house_prices.csv")
#
#checking for the null value
df.isnull().sum()

#get the discription 
df.describe()

Area (sqft)    Bedrooms  Age of House          Price
count   100.000000  100.000000    100.000000     100.000000
mean   2518.550000    2.620000     14.470000  272485.980000
std     855.173671    1.089713      9.072608  129276.577411
min    1021.000000    1.000000      1.000000   52869.000000
25%    1809.000000    2.000000      6.000000  168789.750000
50%    2434.000000    3.000000     16.000000  255488.500000
75%    3273.000000    3.250000     23.000000  383900.500000
max    3945.000000    4.000000     29.000000  494623.000000

#Discription of sample dataset
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>Area (sqft)</th>
      <th>Bedrooms</th>
      <th>Age of House</th>
      <th>Price</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>count</th>
      <td>100.000000</td>
      <td>100.000000</td>
      <td>100.000000</td>
      <td>100.000000</td>
    </tr>
    <tr>
      <th>mean</th>
      <td>2518.550000</td>
      <td>2.620000</td>
      <td>14.470000</td>
      <td>272485.980000</td>
    </tr>
    <tr>
      <th>std</th>
      <td>855.173671</td>
      <td>1.089713</td>
      <td>9.072608</td>
      <td>129276.577411</td>
    </tr>
    <tr>
      <th>min</th>
      <td>1021.000000</td>
      <td>1.000000</td>
      <td>1.000000</td>
      <td>52869.000000</td>
    </tr>
    <tr>
      <th>25%</th>
      <td>1809.000000</td>
      <td>2.000000</td>
      <td>6.000000</td>
      <td>168789.750000</td>
    </tr>
    <tr>
      <th>50%</th>
      <td>2434.000000</td>
      <td>3.000000</td>
      <td>16.000000</td>
      <td>255488.500000</td>
    </tr>
    <tr>
      <th>75%</th>
      <td>3273.000000</td>
      <td>3.250000</td>
      <td>23.000000</td>
      <td>383900.500000</td>
    </tr>
    <tr>
      <th>max</th>
      <td>3945.000000</td>
      <td>4.000000</td>
      <td>29.000000</td>
      <td>494623.000000</td>
    </tr>
  </tbody>
</table>
</div>